infrastructure/sentry-subgraph/src/referee.ts

```diff
@@ -669,3 +672,36 @@ export function handleBulkRewardsClaimed(event: BulkRewardsClaimedEvent): void {
   }
 
 }
+
+export function handleNodeConfirmed(event: NodeConfirmedEvent): void {
+  //subgraph entity id is nodeNum since its already unique
+  const nodeConfirmedEvent = new NodeConfirmation(
+    event.params.nodeNum.toString()
+  )
+  nodeConfirmedEvent.nodeNum = event.params.nodeNum;
+  nodeConfirmedEvent.blockHash = event.params.blockHash;
+  nodeConfirmedEvent.sendRoot = event.params.sendRoot;
+
+  //constructing confirmHash exactly how its built in the smart contract (RollupLib.confirmHash)
+  let blockHashBytes = nodeConfirmedEvent.blockHash;
+  let sendRootBytes = nodeConfirmedEvent.sendRoot;
+
+  if (!blockHashBytes || !sendRootBytes) {
+    log.warning("Failed to assign null event value to variable", []);
+    return;
+  }
+
+  //confirmData = keccak256(blockHash + sendRoot)
+  let concatenatedHexStr = blockHashBytes.concat(sendRootBytes).toHexString();
+  let concatenatedByteArray = ByteArray.fromHexString(concatenatedHexStr);
+  let confirmHashByteArray = crypto.keccak256(concatenatedByteArray);
+  let confirmHashHexStr = confirmHashByteArray.toHexString();
+
+  //assign confirm data
+  nodeConfirmedEvent.confirmData = confirmHashHexStr;
+
+  //temporarily set challenge to placeholder value, set correct value in challenge handler
+  nodeConfirmedEvent.challenge = null;
```

Comment:
I don't think thats needed right ? 
Just confirming, no change requested.

Reply:
I think you may be right - its likely already null to begin with.